units/final.target

```diff
@@ -8,7 +8,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Final Step
+Description=Late Boot Services
```

Comment:
Is this not about shutdown rather than boot?  (I guess shutdown could be considered very late boot!) 

https://github.com/systemd/systemd/blob/main/man/systemd.special.xml#L285:
>A special target unit that is used during the shutdown logic and may be used to pull in late services after all normal services are already terminated and all mounts unmounted.

Reply:
Indeed. I guess "Late Shutdown Services" would be better. Care to file a PR?

Reply:
See https://github.com/systemd/systemd/pull/20112